SonarQube.Client/Models/ServerSentEvents/ClientContract/IIssueChangedServerEvent.cs

```diff
@@ -37,19 +38,23 @@ public interface IIssueChangedServerEvent : IServerEvent
 
     public class IssueChangedServerEvent : IIssueChangedServerEvent
     {
-        public IssueChangedServerEvent(string projectKey, bool isResolved, IBranchAndIssueKey[] branchAndIssueKeys)
+        public IssueChangedServerEvent(string projectKey, bool isResolved, BranchAndIssueKey[] issues)
```

Comment:
I was getting deserialization errors that `BranchAndIssueKeys` (the property) is an interface, and found that this works: if you name the CTOR argument the same name as the json property and make it of the concrete type.

Reply:
👍